packages/sass.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Sass < Package
+  description 'A Dart implementation of Sass. Sass makes CSS fun again.'
+  homepage 'https://sass-lang.com/'
+  version '1.26.10'
+  compatibility 'all'
+  source_url 'https://github.com/sass/dart-sass/archive/1.26.10.tar.gz'
+  source_sha256 '9131be72e3eb6d32265e354d6da9f9162d9b28ed5b4910f03744c31509be894f'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'dart'
+
+  def self.build
+    system "echo '#!/bin/bash' >> sass" 
+    system "echo 'dart #{CREW_PREFIX}/share/dart-sass/bin/sass.dart $@' >> sass"
```

Comment:
Does the `$` need escaped?  What are the contents of sass after install?

Reply:
After installing, here is the content of `sass`. Seems it doesn't need to escape $
```
$ cat /usr/local/bin/sass 
#!/bin/bash
dart /usr/local/share/dart-sass/bin/sass.dart $@
```